src/plays/play-dictionary/PlayDictionary.css

```diff
@@ -0,0 +1,181 @@
+/* enter stlyes here */
+
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.main-container {
```

Comment:
Please use our play specific class name for all the CSS selectors.
like `play-dict-main-container` , ` play-dict-container-input` etc

Reply:
changes for class names and css selectors made as required.